scss/_commercial.scss

```
@@ -0,0 +1,178 @@
+.commercial-navbar {
+  background-color: $orange;
+  margin-top: 0px;
+  .navbar-logo {
+    height: 80%;
+    margin: auto 10px;
+  }
+}
+.commSwiper {
+  width: 100%;
+  height: 100%;
+  margin-top: 100px;
+
+  .swiper-slide {
+    text-align: center;
+    font-size: 18px;
+    background: #fff;
+    transition: filter 1s;
+
+    &-prev {
+      filter: brightness(80%);
+      transition: filter 1s;
+    }
+
+    &-active {
+      & img {
+        filter: brightness(50%);
+        transition: filter 1s;
+
+        &:hover {
+          filter: brightness(40%);
+          transition: filter 1s;
+          cursor: pointer;
+        }
+      }
+
+      & h2 {
```

Ne treba ti &  da bi nestovao
```suggestion
       h2 {
```